RakutenShop/Networking/NetworkRequest.swift

```diff
@@ -26,7 +26,7 @@ extension NetworkRequest {
     var queryParameters: [String: String]? { nil }
     var bodyParameters: [String: String]? { nil }
     var customHeaders: [String: String]? { nil }
-    var authorizationStrategy: AuthorizationStrategy? { nil }
+    var authorizationStrategy: AuthorizationStrategy? { .token }
```

Comment:
question if it has default value, why is it optional?
the same for timeoutInterval